package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "js-search",
-  "version": "1.4.2",
+  "version": "1.4.3",
```

Comment:
Can you revert this bit? I normally wait to bump the package version until right before a commit.

Reply:
Definitely!